soccer/gameplay/evaluation/position.py

```diff
@@ -0,0 +1,48 @@
+import main
+import robocup
+import constants
+import math
+
+'''calculates the distance of a robot to the ball
+def distance(robot):
+	x = abs(main.ball().pos.x - robot.pos.x)
+	y = abs(main.ball().pos.y - robot.pos.y)
+	dist = math.sqrt((x*x) + (y*y))
+	return dist
+'''
+	
+'''returns all robots, as well as their average positions, that are within a given radius of the ball'''
```

Comment:
What does average position mean here, you're returning the average angle around the ball?

Average angle also isn't very good of metric in this regard if you just add the angles and divide by the number of robots. Something at -.99PI and .99PI shouldn't result in an average of 0 PI, it should be PI. I've converted to Cartesian coord, averaged them in the past. This comes out a little better in terms of averaging angle

Reply:
I am returning both pi and 0, and whichever of the two corresponding points (both or neither is also acceptable) is within the field of play are assigned robots to wait at those positions.